### PR TITLE
Create type definitions and stub api functions

### PR DESCRIPTION
**Pull request by crshenx**
Co-authored-by: Logan Figgins <L-Figgins@users.noreply.github.com>

### CODE REVIEW

File: client/src/api/api.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+   * Route /<prefix>/articles/:id
+   * Method GET get one article
+   * @param {string} id - article's id
+   * @return {Promise<Article>} - article object
+   *
+   * */
+  async getArticle(id) {},
+};
+
+// payload  object {}}
```
Comment (L-Figgins) [inline client/src/api/api.js]: please remove comment that was accidentatly left in

Comment (crshenx) [reply client/src/api/api.js]: Thank you! Comment removed and exported default!

Comment (L-Figgins) [reply client/src/api/api.js]: Looks good. Thanks! approving

**Review by L-Figgins — APPROVED** [review]
Looks Good.

**Pull request merged**